### PR TITLE
config: implemented basic validation

### PR DESCRIPTION
**Pull request by sontags**
This implements the subcommand `verify-config` and checks if ...

* the regexp in `singularity.expression` matches with the rendered results of `singularity.expression_template`
* the keys match the matchgroup defined in `singularity.expression`

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Found some fixes!

_P.S. share your ideas, feedbacks or issues with us at https://github.com/fixmie/feedback (this message will be removed after the beta stage)._

File: main.go [hunk 1]
```diff
@@ -164,9 +160,16 @@ func main() {
...
+func exitOnErr(errs ...error) {
+	errNotNil := false
+	for _, err := range errs {
+		if err == nil {
+			continue
+		}
+		errNotNil = true
+		fmt.Fprintf(os.Stderr, "ERROR: %s", err.Error())
+	}
+	if errNotNil {
```
Comment (Juerg-Liechti) [inline main.go:172]: nice done :-)

**Review by Juerg-Liechti — APPROVED** [review]
lgtm

File: main.go (hunk 1 above)
Comment (sontags) [reply main.go:172]: Merci

**Pull request merged**